src/Bicep.Cli.IntegrationTests/DecompileTests.cs

```diff
@@ -24,7 +24,7 @@ private static (IEnumerable<string> outputLines, IEnumerable<string> errorLines,
         {
             var (output, error, result) = TextWriterHelper.InvokeWriterAction((outputWriter, errorWriter) =>
             {
-                var program = new Program(TestResourceTypeProvider.Create(), outputWriter, errorWriter);
```

Comment:
Speaking of the decompiler, is there anything we need to change there to have it ignore the metadata properties? Please could you add a test to assert that metadata gets ignored?